back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -49,51 +49,47 @@
       parameter :review_state, 'Filter by project review status (pending, approved)', required: false
 
       example_request 'List all admin publications' do
+        hidden_project = create(:community_monitor_project)
         expect(status).to eq(200)
-        json_response = json_parse(response_body)
-        expect(json_response[:data].size).to eq 10
-        expect(json_response[:data].map { |d| d.dig(:relationships, :publication, :data, :type) }.count('project')).to eq 8
-        expect(json_response[:data].map { |d| d.dig(:relationships, :publication, :data, :type) }.count('folder')).to eq 2
+        expect(response_data.size).to eq 10
+        expect(response_data.map { |d| d.dig(:relationships, :publication, :data, :type) }.count('project')).to eq 8
+        expect(response_data.map { |d| d.dig(:relationships, :publication, :data, :type) }.count('folder')).to eq 2
+        expect(response_data.pluck(:id)).not_to include(hidden_project.admin_publication.id)
       end
```

Comment:
This is the only test that has actually changed. The rest of the code changes are just changing to use `response_data` helper method